packages/side-nav/test/side-nav-item.test.js

```diff
@@ -292,5 +292,27 @@ describe('side-nav-item', () => {
       toggle.click();
       expect(spy.called).to.be.false;
     });
+
+    describe('target property', () => {
+      it('should set target attribute to the anchor when target is set', async () => {
+        item.target = '_blank';
+        await nextRender();
+        expect(anchor.getAttribute('target')).to.be.equal('_blank');
+      });
+
+      it('should remove target attribute from the anchor when target is removed', async () => {
+        item.target = '_blank';
+        await nextRender();
+        item.target = null;
+        await nextRender();
+        expect(anchor.getAttribute('target')).to.be.not.ok;
+      });
+
+      it('should not set target attribute to the anchor when target is empty string', async () => {
+        item.target = '';
+        await nextRender();
+        expect(anchor.getAttribute('target')).to.be.not.ok;
```

Comment:
From my testing the attribute is still defined for an empty string, the assertion here doesn't work as intended. Would be better to use `hasAttribute` for checking the attribute existence IMO.

If we want to avoid defining the attribute it requires further changes. Don't see why it would be necessary though. So maybe just remove the test case.

Reply:
Yeah, you're right. Based on this comment, I also changed the assertion in the previous test to use `hasAttribute` instead.